alternator/executor.cc

```diff
@@ -3470,7 +3470,11 @@ void executor::describe_single_item(const cql3::selection::selection& selection,
                     // names are unique so add() makes sense
                     rjson::add_with_string_name(item, attr_name, std::move(v));
                 } else if (item_length_in_bytes) {
-                    (*item_length_in_bytes) += value_cast<bytes>(entry.second).length() - 1;
+                    auto len = value_cast<bytes>(entry.second).length();
+                    // BUG FIX 9: Prevent unsigned integer underflow
+                    if (len > 0) {
+                        (*item_length_in_bytes) += len - 1;
+                    }
```

Comment:
CC @amnonh who wrote this code (commit b888ed84f78):

I think this is not a practical bug, at least not specific to this code: We would never have a single item sized over 2GB, and if we do, wrapping around the length calculation is the least of our worries. Moreover, Scylla's "bytes" and "sstring" both use 32-bit lengths, so if we allow 2GB strings our entire code will break in many places. In fact, I wonder if we ever tested this. For example, we can incrementally add attributes to an Alternator item, and it will grow and grow until it's 2GB. I'm pretty sure we wouldn't be able to read it, but would something (compaction?) fail earlier? In any case, having a single 2GB item will ruin any hope for good latency.